Makefile

```diff
@@ -30,7 +30,7 @@ dev_database :
 	${MANAGE} loaddata amy/autoemails/fixtures/templates_triggers.json
 	${MANAGE} fake_database
 	${MANAGE} createinitialrevisions
-	${MANAGE} create_superuser
+	${MANAGE} createsuperuser
```

Comment:
This wasn't broken. We have our own command `create_superuser` in `amy/workshops/management/commands/create_superuser.py`.

Reply:
Oh I didn't realize. I thought it was a typo. Thanks for letting me know.